src/SFA.DAS.ReferenceData.PublicSectorOrgs.WebJob/Updater/PublicOrgsUpdater.cs

```diff
@@ -20,17 +20,26 @@ public class PublicOrgsUpdater : IPublicOrgsUpdater
     {
         private readonly IArchiveDownloadService _archiveDownloadService;
         private readonly INhsDataUpdater _nhsDataUpdater;
+        private readonly IPublicSectorOrganisationDatabaseUpdater _publicSectorOrganisationDatabaseUpdater;
+        private readonly IPublicSectorOrganisationHtmlScraper _publicSectorOrganisationHtmlScraper;
+        private readonly IJsonManager _jsonManager;
         private readonly ILog _logger;
         private readonly ReferenceDataApiConfiguration _configuration;
         private readonly string _workingFolder;
         private readonly string _fileName = "publicsectorclassificationguidelatest";
         private readonly string _jsonContainerName = "sfa-das-reference-data";
```

Comment:
This is marked by resharper as "never used"

Reply:
not sure why that is as they definitely are in use, line 75 and 76 call methods on the _jsonManager member variable.

Reply:
_jsonContainerName isn't. Just that one